http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitorSpec.scala

```diff
@@ -234,7 +232,7 @@ class LeoMetricsMonitorSpec extends AnyFlatSpec with LeonardoTestSuite with Test
         .unsafeRunSync()(IORuntime.global)
     // An up and a down metric for 7 services: 2 cbases, cbas-ui, cromwell, cromwell-reader, cromwell-runner, galaxy
     test.size shouldBe 14
```

Comment:
Weird that the size of this is still 14... I would think it should be 12 without cbas-ui

Reply:
Hmm good point. I will take a look at this.

Reply:
Thank you for pointing this out. I had missed removing a cbas-ui reference from the test file.